client/controller/client/http.go

```diff
@@ -111,7 +111,7 @@ func checkForErrors(res *http.Response, body string) error {
 	bodyMap := make(map[string]interface{})
 
 	if err := json.Unmarshal([]byte(body), &bodyMap); err != nil {
-		return err
+		return errors.New("\n" + body + "\n" + res.Status + "\n")
```

Comment:
The python client was reversed, which would show the status then the raw HTML. Could we change it to display it in that order or is there a reason for this ordering?